### ISSUE TITLE
Error logged twice

### ISSUE THREAD

**Issue by twoeo** [nomiddlename/log4js-node#389]
### Code

```
var log4js = require('log4js');
log4js.configure({
    appenders: [
        {
            type: 'console'
        },
        {
            type: 'file',
            filename: 'logs/access.log',
            maxLogSize: 1024,
            backups:4
        }
    ],
    replaceConsole: true
});
var logger = log4js.getLogger();

try{
    throw new Error("My Error")
}
catch(e){
    logger.error(e)
}
```

### Expect 

Error logged once

### Actual

```
[2016-05-24 10:54:06.114] [ERROR] [default] - Error: My Error
    at Object.<anonymous> (E:\Codes\test\log4js-test\index.js:53:11)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:456:32)
    at tryModuleLoad (module.js:415:12)
    at Function.Module._load (module.js:407:3)
    at Function.Module.runMain (module.js:575:10)
    at startup (node.js:159:18)
    at node.js:444:3
Error: My Error
    at Object.<anonymous> (E:\Codes\test\log4js-test\index.js:53:11)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:456:32)
    at tryModuleLoad (module.js:415:12)
    at Function.Module._load (module.js:407:3)
    at Function.Module.runMain (module.js:575:10)
    at startup (node.js:159:18)
    at node.js:444:3

```

**Error: My Error was logged twice**

**Comment by nomiddlename** [comment]
Looks like this was a change introduced in node v6, Error's string representation now includes the stack, which it doesn't in previous versions. I'll see if I can work around it.

**Comment by nomiddlename** [comment]
Fix release in log4js 0.6.37
